### PR TITLE
Blueprint: Resilient apps use extra Machines

### PR DESCRIPTION
**Pull request by andie787**
### Summary of changes

A new blueprint for "High availability with Machines" on Fly.io.

### Preview

https://aa-docs-2.fly.dev/docs/blueprints/extra-machines-resilient-apps/

### CODE REVIEW

File: blueprints/extra-machines-resilient-apps.html.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+title: Resilient apps use extra Machines
+layout: docs
+nav: firecracker
+---
+
+Fly Machines are fast-launching VMs; they're the compute of the Fly.io platform. Every Machine is assigned to a single physical host. This helps keep `start` operations consistently fast, but comes at the cost of Machine availability when something goes wrong on a single host.
```
Comment (catflydotio) [inline blueprints/extra-machines-resilient-apps.html.md:7]: I'm almost tempted to leave out the thing about start operations and just say 

```
Every Fly Machine runs on a specific physical host. If that host fails, that Machine simply becomes unavailable; it does not automatically get rescheduled on another host. 

To make your app resilient to single-host failure, create at least two Machines. We have two platform features that will wake up the second Machine when needed.
```

(In place of the paragraph after this too)

File: blueprints/extra-machines-resilient-apps.html.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+title: Resilient apps use extra Machines
```
Comment (catflydotio) [inline blueprints/extra-machines-resilient-apps.html.md:2]: multiple Machines?

File: blueprints/extra-machines-resilient-apps.html.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+Reasons to set up extra Machines:
+
+- Your app won't fail when a single host fails because another Machine will start up to take over app requests or tasks.
+- If your app has a lot of users, or bursts of high usage, then the Fly Proxy can load balance requests and automatically stop and start Machines based on traffic to your app. 
+- When Machines aren't running, you only pay for rootfs, not CPU and RAM. Machine rootfs is [cheap](/docs/about/pricing/#stopped-fly-machines), like 18 cents a month for an average Elixir app that uses 1.2 GB of rootfs.
+
+You can add [extra Machines](#extra-machines-for-apps-that-receive-requests) for Fly Proxy to start and stop as needed, which is great for apps that have built-in replication or that don't share data.
+
+For worker Machines that don't accept external requests, [create a standby Machine](#create-a-standby-machine), stopped and ready to take over in case the original Machine becomes unavailable.
```
Comment (catflydotio) [inline blueprints/extra-machines-resilient-apps.html.md:19]: The thrust of this is that if the Machine can't be woken by the Fly Proxy you can use a standby Machine instead, right?

File: blueprints/extra-machines-resilient-apps.html.md (hunk 1 above)
Comment (andie787) [reply blueprints/extra-machines-resilient-apps.html.md:7]: Thanks. I took some of this and mixed it with that to come up with a better intro. :)

File: blueprints/extra-machines-resilient-apps.html.md (hunk 3 above)
Comment (andie787) [reply blueprints/extra-machines-resilient-apps.html.md:19]: Yes. I am removing this from the current location and into the respective section. I will reword it there.

**Pull request merged**